.github/workflows/ci_windows_x64_msvc.yml

```diff
@@ -15,24 +15,34 @@ on:
 concurrency:
   # A PR number if a pull request and otherwise the commit hash. This cancels
   # queued and in-progress runs for the same PR (presubmit) or commit
-  # (postsubmit). The workflow name is prepended to avoid conflicts between
-  # different workflows.
+  # (postsubmit)
   group: ${{ github.workflow }}-${{ github.event.number || github.sha }}
   cancel-in-progress: true
 
 jobs:
   windows_x64_msvc:
-    runs-on: windows-2022
-    defaults:
-      run:
-        shell: bash
+    runs-on: azure-windows-scale
     env:
-      BUILD_DIR: build-windows
+      BASE_BUILD_DIR_POWERSHELL: C:\mnt\azure\b
+      SCCACHE_AZURE_CONNECTION_STRING: "${{ secrets.AZURE_CCACHE_CONNECTION_STRING }}"
+      SCCACHE_AZURE_BLOB_CONTAINER: ccache-container
+      SCCACHE_CCACHE_ZSTD_LEVEL: 10
+      SCCACHE_AZURE_KEY_PREFIX: "ci_windows_x64_msvc"
```

Comment:
FYI looks like we'll need some extra setup to use sccache on Windows beyond just running the build_all.sh script.

Linux uses `source ./build_tools/cmake/setup_sccache.sh`. Can write a similar script for Windows or work something back into `build_all.sh`.

Fine to keep this here, but it does make it seem like caching is expected to work, when I think it won't. Something to look into as part of https://github.com/iree-org/iree/issues/18557